### PR TITLE
fixed typo 7/28

### PR DESCRIPTION
**Pull request by stluo**
# first open source pull

Any chance of an extra credit for paying attention? Maybe +0.5%? 

Cheers

### CODE REVIEW

File: lectures/L07-slides.tex [hunk 1]
```diff
@@ -219,7 +219,7 @@
 
 \begin{frame}
 \frametitle{Another Mistake}
-\textit{Great Peace Shipping Ltd v Tsavliris Salvage�(International) Ltd.,} 2002.
+\textit{Great Peace Shipping Ltd v Tsavliris SalvageÊ(International) Ltd.,} 2002.
```
Comment (jzarnett) [inline lectures/L07-slides.tex]: If modifying this line it would be good to remove the extraneous Ê in the title.

File: lectures/L07-slides.tex [hunk 2]
```diff
@@ -105,7 +105,7 @@
 
 \alert{Common Mistake}: the parties understand each other but both are mistaken about certain facts.
 
-\alert{Mutual Mistake}: one party is thinking one thing, the other is thinking another, and they are unaware they they are misunderstanding each other.
+\alert{Mutual Mistake}: one party is thinking one thing, the other is thinking another, and they are unaware that they are misunderstanding each other.
```
Comment (jzarnett) [inline lectures/L07-slides.tex:108]: Good find, thank you.

**Review by jzarnett — CHANGES_REQUESTED** [review]
Sadly, the syllabus does not permit extra credit for finding items to correct in the lecture notes, but you nevertheless receive my appreciation. :)

**Review by stluo — COMMENTED** [review]
in trying to fix something, i've accidentally introduced another mistake, definition of my life.

**Comment by stluo** [comment]
Note: thanks for this experience, never seen/used this functionality of git before

**Comment by jzarnett** [comment]
Closed means I don't merge it; you probably wanted to leave it open :)

**Review by jzarnett — APPROVED** [review]

**Pull request merged**